src/main/java/bio/terra/cli/command/shared/WsmBaseCommand.java

```diff
@@ -0,0 +1,52 @@
+package bio.terra.cli.command.shared;
+
+import bio.terra.cli.app.utils.VersionCheckUtils;
+import bio.terra.cli.businessobject.Context;
+import bio.terra.cli.businessobject.User;
+import bio.terra.cli.command.Main;
+import bio.terra.cli.utils.Logger;
+import bio.terra.cli.utils.UserIO;
+import java.io.PrintStream;
+import org.slf4j.LoggerFactory;
+
+public abstract class WsmBaseCommand extends BaseCommand {
```

Comment:
```
/**
 * This class prints a warning if CLI version is too old for WSM version.
 * 
 * Commands which call WSM should extend this instead of BaseCommand. (This way, if WSM happens to be down, commands that don't call WSM will still succeed.)
 */
```
In general, if a class isn't obvious, there should be a comment. :)